core/baseline-core-v1.0-psd01.md

```diff
@@ -2309,47 +2307,249 @@ New Agreement State = Old Agreement State + Agreed upon New State Object + Works
+2. For each workstep in the list, validate that the workstep identifiers are not duplicate.
+3. Create a new workstep within the BPI and verify that it's identifier does not exist in the list of other workstep identifiers.
+4. Attempt to create a new workstep within the BPI with the same identifier as an existing workstep and verify that it is rejected.
+
+Test Passing criteria:
+
+* Test step 2 passes if all worksteps in the list have a unique identifier within the BPI.
+* Test step 3 passes if the new workstep is assigned a unique identifier.
+* Test step 4 passes if the attempt to create a workstep with the same identifier as an existing workstep is rejected.
+
 #### **[R218]** 
 A workstep MUST be updatable.
 
+[[R218]](#r218) Testability:
+
+Preconditions:
+
+* A BPI test instance is set up and operational.
+* A workstep exists in the BPI system.
+
+Test Steps:
+
+1. Access the workstep in the BPI system.
+2. Attempt to update the workstep with new information.
+3. Save the changes to the workstep.
+4. AAccess the workstep in the BPI system again.
+5. Verify that the new workstep information displays correctly
+
+Test Passing Criteria:
+
+* The changes to the workstep are saved successfully and can be viewed when accessing the workstep again.
+
 #### **[R219]** 
-A workstep instance MUST NOT be updated while the workstep is being executed by the BPI.
+A workstep instance MUST NOT be updated while the workstep instance is being executed by the BPI.
 
 *This ensures that no breaking changes with potentially significant negative business impact are introduced while a workstep instance is being executed.*
 
+[[R219]](#r219) Testability:
+
+Preconditions:
+
+* A BPI test instance is set up and operational.
+* A BPI with at least one workstep defined.
+* A workstep instance that is currently being executed by the BPI.
+
+Test steps:
+
+1. Get the identifier of the workstep instance that is currently being executed by the BPI.
+2. Attempt to update the workstep instance while it is being executed by the BPI.
+3. Verify that the workstep instance was not updated and that an appropriate error message is returned.
+4. Wait for the workstep instance to complete execution.
+5. Verify that the workstep instance can now be updated.
+
+Test Passing criteria:
+
+* Step 3 must return an appropriate error message indicating that the workstep instance cannot be updated while it is being executed by the BPI.
+* Step 5 must successfully update the workstep instance.
+
 #### **[R220]** 
 A workstep MUST be versioned within a BPI.
 
 *Note that versions of the same workstep do not have to be compatible with one another.* 
 
+[[R220]](#r220) Testability:
+
+Preconditions:
+
+* A BPI test instance is set up and operational.
+* A BPI has been created and contains at least one workstep.
+* The BPI allows for versioning of worksteps.
+
+Test Steps:
+
+1. Create a new workstep within the BPI.
+2. Verify that the workstep has been assigned a version number (e.g., 1.0).
```

Comment:
what's wrong with alphanumeric identifiers? Nothing in the requirement specifies a number